scene/gui/tree.cpp

```diff
@@ -314,7 +308,7 @@ void TreeItem::set_text(int p_column, String p_text) {
 		}
 		cells.write[p_column].step = 0;
 	} else {
-		// Don't auto translate if it's in string mode and editable, as the text can be changed to anything by the user.
+		// Don't TreeItem* translate if it's in string mode and editable, as the text can be changed to anything by the user.
```

Comment:
```suggestion
		// Don't auto translate if it's in string mode and editable, as the text can be changed to anything by the user.
```
Please don't just use search/replace without checking what you're replacing